src/pal_park.c

```diff
@@ -0,0 +1,281 @@
+#include "global.h"
+#include "field_system.h"
+#include "gf_rtc.h"
+#include "game_stats.h"
+#include "save_arrays.h"
+#include "math_util.h"
+#include "battle_setup.h"
+#include "unk_02054648.h"
+#include "metatile_behavior.h"
+#include "constants/game_stat.h"
+#include "pal_park.h"
+#include "arc/ppark.naix"
+
+struct PalParkMon {
+    u16 species;
+    u8 encounter_tile;
```

Comment:
Change to camelCase for this plus other struct elements

Reply:
The snake says: *HISSSSSSSS*